ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -1,249 +1,24 @@
-const DATASET_TYPE_VARIANT_MITO = `${DATASET_TYPE_MITO_CALLS},${DATASET_TYPE_SNV_INDEL_CALLS}`
-const DATASET_TYPE_VARIANT_SV = `${DATASET_TYPE_SV_CALLS},${DATASET_TYPE_SNV_INDEL_CALLS}`
-
-const NO_HGMD_PANEL_PROPS = {
-  headerProps: {
-    ...HGMD_PATHOGENICITY_PANEL.headerProps,
-    inputProps: JsonSelectPropsWithAll(PATHOGENICITY_FILTER_OPTIONS, ANY_PATHOGENICITY_FILTER),
-  },
-  fields: PATHOGENICITY_FIELDS,
-}
-
-const ANNOTATION_SECONDARY_NAME = 'annotations_secondary'
-const SV_GROUPS_NO_NEW = SV_GROUPS.filter(name => name !== VEP_GROUP_SV_NEW)
-const ANNOTATION_SECONDARY_PANEL = {
-  ...ANNOTATION_PANEL,
-  headerProps: { ...ANNOTATION_PANEL.headerProps, title: 'Annotations (Second Hit)' },
-  name: ANNOTATION_SECONDARY_NAME,
-  helpText: (
-    <span>
-      Apply a secondary annotation filter to compound heterozygous pairs. All pairs of variants will include exactly one
-      variant that matches the above annotation filter and one variant that matches this secondary annotation filter.
-      <br />
-      For recessive searches, homozygous and X-linked recessive variants will be filtered using the above main
-      annotations filter.
-    </span>
-  ),
-  fieldLayout: annotationFieldLayout(
-    [HIGH_MODERATE_IMPACT_GROUPS, CODING_OTHER_IMPACT_GROUPS, SV_GROUPS_NO_NEW],
-  ),
-}
-
-const PANELS = [
-  INHERITANCE_PANEL,
-  HGMD_PATHOGENICITY_PANEL,
-  ANNOTATION_PANEL,
-  ANNOTATION_SECONDARY_PANEL,
-  IN_SILICO_PANEL,
-  FREQUENCY_PANEL,
-  LOCATION_PANEL_WITH_GENE_LIST,
-  QUALITY_PANEL,
-]
-
-const DATASET_TYPE_PANEL_PROPS = {
-  [DATASET_TYPE_SNV_INDEL_CALLS]: {
-    [ANNOTATION_PANEL.name]: {
-      fieldLayout: annotationFieldLayout(
-        [HIGH_MODERATE_IMPACT_GROUPS, CODING_OTHER_IMPACT_GROUPS, ANNOTATION_OVERRIDE_GROUPS],
-      ),
-    },
-    [ANNOTATION_SECONDARY_NAME]: {
-      fieldLayout: annotationFieldLayout([HIGH_MODERATE_IMPACT_GROUPS, CODING_OTHER_IMPACT_GROUPS]),
-    },
-    [IN_SILICO_PANEL.name]: {
-      fieldLayout: inSilicoFieldLayout(NO_SV_IN_SILICO_GROUPS),
-    },
-    [FREQUENCY_PANEL.name]: {
-      fields: SNP_FREQUENCIES,
-    },
-    [QUALITY_PANEL.name]: {
-      fields: SNP_QUALITY_FILTER_FIELDS,
-    },
-  },
-  [DATASET_TYPE_VARIANT_SV]: {
-    [FREQUENCY_PANEL.name]: {
-      fields: SNP_FREQUENCIES.concat(SV_FREQUENCIES),
-    },
-    [QUALITY_PANEL.name]: {
-      fields: SNP_QUALITY_FILTER_FIELDS.concat(SV_QUALITY_FILTER_FIELDS),
-    },
-  },
-}
-
-DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_MITO] = {
-  ...DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_SNV_INDEL_CALLS],
-  [FREQUENCY_PANEL.name]: {
-    fields: SNP_FREQUENCIES.concat(MITO_FREQUENCIES),
-  },
-  [QUALITY_PANEL.name]: {
-    fields: SNP_QUALITY_FILTER_FIELDS.concat(MITO_QUALITY_FILTER_FIELDS),
-  },
-}
-
-const HAS_HGMD = true
-const NO_HGMD = false
-const HAS_ANN_SECONDARY = true
-const NO_ANN_SECONDARY = false
-
-const PANEL_MAP = [ALL_DATASET_TYPE, DATASET_TYPE_VARIANT_MITO, DATASET_TYPE_VARIANT_SV,
```

Comment:
this is some of the only code in this PR thats not just copy-pasted around. Instead of maintaining this really complex nested map of different panel constants dependent on dataset type, inheritance mode and HGMD we have now moved all of that conditional logic directly into the panel rendering